site/routes/domain.vue

```diff
@@ -47,6 +47,18 @@
                 :def="def"/>
         </template>
 
+        <template v-if="errorsByCategory">
+            <h2 id="errors">Errors</h2>
+            <div
+                v-for="category of Object.keys(errorsByCategory)"
+                class="domain__errors">
+                <h3 class="domain__errors__subheader"> {{ category }} </h3>
```

Comment:
A-ha! 😄 We use special BEM notation, which is `block-name__element-name--modifier-name`, so the above classname should look like `domain__errors-subheader`